webapp/src/Service/CheckConfigService.php

```diff
@@ -60,14 +61,20 @@ class CheckConfigService
      */
     protected $debug;
 
+    /**
+     * @var $passwordEncoder
```

Comment:
this @var is quite useless.

Reply:
All of these "@Var" should be removed right? (in that section)